### PR TITLE
Add documentation for the oc rsync command

### PR DESCRIPTION
**Pull request by csrwng**
Documents the new 'oc rsync' command.

### CODE REVIEW

**Comment by csrwng** [comment]
@mfojtik @adellape ptal

File: dev_guide/copy_files_to_container.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+$ oc rsync /home/user/source devpod1234:/src
+----
+====
+
+== Requirements
+
+The `oc rsync` command uses the local `rsync` command if present on the client's machine. This requires
+that the remote container also have the `rsync` command.  If `rsync` is not found locally or in the remote 
+container, then a tar archive will be created locally and sent to the container where `tar` will be used to
+extract the files. If `tar` is not available in the remote container, then the copy will fail.
```
Comment (mfojtik) [inline dev_guide/copy_files_to_container.adoc]: What if I have rsync locally but not in the container? Also maybe worth mention that 'rsync' is part of centos (debian?) base image, so people will no freak out installing rsync :-)

**Comment by mfojtik** [comment]
LGTM

File: dev_guide/copy_files_to_container.adoc (hunk 1 above)
Comment (csrwng) [inline dev_guide/copy_files_to_container.adoc]: If rsync is available locally and not in the container, we will fall back to tar. That's what I thought I said up there, but maybe should make it more explicit?

**Comment by adellape** [comment]
:+1:

**Comment by vikram-redhat** [comment]
Doesn't require a followup.

**Pull request merged**